api/utils/keypaths/keypaths.go

```diff
@@ -226,16 +228,32 @@ func KubeCertPath(baseDir, proxy, username, cluster, kubename string) string {
 	return filepath.Join(KubeCertDir(baseDir, proxy, username, cluster), kubename+fileExtTLSCert)
```

Comment:
I noticed some of the `Database*` functions defined above have comments referring to "kube directory". That's probably a mistake, would you mind fixing the comments as part of this PR?